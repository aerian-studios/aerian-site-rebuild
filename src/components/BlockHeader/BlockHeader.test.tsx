import * as React from "react";

import * as renderer from "react-test-renderer";

import { BlockHeader } from "./index";

describe("BlockHeader", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<BlockHeader title="foo" text="bar" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
