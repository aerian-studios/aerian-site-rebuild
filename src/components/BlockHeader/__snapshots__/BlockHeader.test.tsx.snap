// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockHeader renders correctly 1`] = `
<div
  className=" myClass"
>
  <h2 />
  <p />
</div>
`;
